### ISSUE TITLE
Windows for Workgroups 3.11 won't work

### ISSUE THREAD

**Issue by jadasse** [86Box/86Box#1206]
After using the guide to install windows, for some reason, when I try to type win, either it hangs on the logo, and 86box stops working, or it passes the logo, but I'm unable to access the mouse on the login/logon screen, after a while, it takes me back to command. What is going on?

**Comment by jadasse** [comment]
I tried reinstalling Windows, but the problem still occurs

**Comment by ppgrainbow** [comment]
Can you provide the 86Box.cfg file and which 86Box build was responsible for the problem?

**Comment by jadasse** [comment]
[General]
video_fullscreen_first = 0
dpi_scale = 0
confirm_save = 0
confirm_exit = 0
confirm_reset = 0

[Machine]
machine = r418
cpu_family = i486dx
cpu_speed = 33333333
cpu_multi = 1
mem_size = 16384
cpu_use_dynarec = 1
time_sync = local
fpu_type = internal
cpu = 7

[Video]
gfxcard = stealth64v_pci

[Input devices]
mouse_type = msserial

[Storage controllers]
hdc = internal

[Sound Blaster 16]
base = 0220
base401 = 0330
irq = 5
dma = 1
dma16 = 5
opl = 1
receive_input = 1
receive_input401 = 1

[System MIDI]
midi = 1

[Realtek RTL8029AS]
mac = 5c:cd:4f

[Sound]
sndcard = sb16
midi_device = system_midi

[Network]
net_type = slirp
net_card = ne2kpci

[Hard disks]
hdd_01_parameters = 63, 10, 1013, 0, ide
hdd_01_fn = Windows for Workgroups 3.11
hdd_01_ide_channel = 0:0

[Floppy and CD-ROM drives]
fdd_01_type = 35_2hd
fdd_02_type = 525_2hd
cdrom_01_speed = 12
cdrom_01_parameters = 1, atapi
cdrom_01_ide_channel = 0:1
fdd_01_fn = C:\Users\dlada\Documents\Windows for Workgroups 3.11\Diamond Stealth 64 VRAM (S3 Vision964) (Windows 3.1) (3.5).img
cdrom_01_host_drive = 200
cdrom_01_image_path = C:\Users\dlada\Documents\Windows for Workgroups 3.11\Microsoft Windows for Workgroups 3.11 (OEM) (3.5-1.44mb)\wfw311installation.iso

[Ports (COM & LPT)]
lpt1_device = postscript
lpt2_enabled = 1
lpt2_device = dss

3.0.0.3765

**Comment by OBattler** [comment]
It could be an IRQ conflict between the LPT port and the sound card. My installation on pretty much the same configuration except that it uses the Intel Advanced/EV and the Pentium 75, works fine.

**Comment by jadasse** [comment]
No, I'm using the Haswell build (cpu optimation)
Before you watch, one attempt at recording it was really funny, windows was actually booting and operating correctly! Sometimes it works, sometimes it doesn't.
https://user-images.githubusercontent.com/71288109/103905999-56389a80-50cd-11eb-8481-2cd5b3339749.mp4
BTW near the end I tried to hard-reset it, but 86box crashes :(
nvr: https://mega.nz/folder/i6wjFYab#AvUVCCt4peGeQWh08TuISw
hdi: https://mega.nz/file/7j4hDSyY#NxXZy9LIpZz03EQtfiLL_Zc4vGjRGA14F0CM9toi-2c

**Comment by jadasse** [comment]
Sorry for the small video size, I used OBS to record it

**Comment by jadasse** [comment]
ill improve

**Comment by jadasse** [comment]
Hello?

**Comment by jadasse** [comment]
hello?

**Comment by jadasse** [comment]
Hello?????

**Comment by daviunic** [comment]
We're looking into this, once there's an update you'll see it on this issue.

**Comment by jadasse** [comment]
Ok thx!

**Comment by jadasse** [comment]
I was waiting if someone saw my uploads

**Comment by jadasse** [comment]
I assume there's unfinished progress, right? It's been a week.

**Comment by jadasse** [comment]
It's been two weeks

**Comment by OBattler** [comment]
I saw your uploads, I just haven't gotten around to looking the VM yet.

**Comment by jadasse** [comment]
oh ok

**Comment by jadasse** [comment]
I wanted to make a new VM to fix the problem. So I extracted the IMG HDD to the folder and deleted the windows folder because I thought that would fix the problem of Window's setup "not having enough available conventional memory." However, this didn't work, so I just started over from the beginning of the guide. After finishing up the installation, I was still persistent in not moving the mouse and closing automatically; however, I'll assume what's going on. Whenever my mouse isn't trapped in the VM and then click in, the mouse works as it should! However, when my mouse is already in the VM, I can't do anything, so I think I got this; thanks!
